src/component/actionbar/index.tsx

```diff
@@ -21,6 +21,9 @@ function ActionBar(props: ActionBarProps) {
       textContainer={{
         render: () => <ComposeEditor setHasContent={setHasContent} />,
       }}
+      sendButton={{
+        onPress: () => alert("pressed!"),
```

Comment:
I am wondering why we should do this. What is the intention behind this? I feel like I am reviewing something that is not ready.